src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3514,6 +3514,12 @@ private BoundNode BindConstructorBody(ConstructorDeclarationSyntax constructor,
                 && thisInitializer
                 && ContainingType.IsDefaultValueTypeConstructor(initializer);
 
+            if (skipInitializer &&
```

Comment:
>skipInitializer [](http://example.com/codeflow?start=16&length=15)

The fact that ```skipInitializer``` depends on ```includesFieldInitializers``` looks suspicious. I think it is technically possible to have a primary constructor and no initializers. It would be good trying to construct a scenario like that. Consider instead using condition ```thisInitializer && ContainingType.IsDefaultValueTypeConstructor(initializer)```, I think that would exactly reflect the condition that design requires us to check.